api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -386,6 +387,26 @@ public RawlsWorkspaceACLUpdateResponseList updateWorkspaceACL(
         (context) -> workspacesApi.updateACL(aclUpdates, workspaceNamespace, firecloudName, false));
   }
 
+  /**
+   * Updates the Access Control List (ACL) for a specified workspace. If Published: Grant Reader
+   * access to all users in the groupEmail If Unpublished: Grant No access to all users in the
+   * groupEmail
+   *
+   * @param workspaceNamespace the Namespace (Terra Billing Project) of the Workspace to modify
+   * @param firecloudName the Terra Name of the Workspace to modify
+   * @param groupEmail the email of the group to update
+   * @return
+   */
+  @Override
+  public void updatePublishWorkspaceACL(
```

Comment:
something feels not quite right about the name here.  But it's close.  how about `updateWorkspaceAclForPublishing()` ?